config/locales/transcribe/transcribe-es.yml

```diff
@@ -59,7 +59,6 @@ es:
       save_changes: Guardar Cambios
       save_to_incomplete_tooltip: Guardar una página incompleta
       save_to_needs_review_tooltip: SGuardar tus cambios
```

Comment:
It looks like there is a typo in this line.  Let's replace `SGuardar` with `Guardar` to match the others.